os/board/rtl8721csm/src/component/common/drivers/wlan/realtek/include/wifi_structures.h

```diff
@@ -225,17 +225,19 @@ typedef struct ieee80211_frame_info{
 
 #if defined(CONFIG_UNSUPPORT_PLCPHDR_RPT) && CONFIG_UNSUPPORT_PLCPHDR_RPT
 typedef struct rtw_rx_info {
-	uint16_t length;	//length without FCS
-	uint8_t filter;		// 2: 2T rate pkt; 3: LDPC pkt
+	unsigned short length;	// length without FCS
+	unsigned char filter;		// 1: HT-20 2T and not LDPC pkt; 2: HT-40 2T and not LDPC pkt; 3: LDPC pkt
 	signed char rssi;	//-128~-1
+	unsigned short channel;	// channel which this pkt in

```

Comment:
Please modify `buuut` to `but`. And you want to use `actual` instead of `antual`?

Reply:
Resolved on [daa8fd5fd2cb81556c031b48f6eb3222ba653f39]